src/main/java/com/querydsl/couchbase/AbstractCouchbaseQuery.save

```diff
@@ -0,0 +1,327 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
+
+package com.querydsl.couchbase;
```

Comment:
Missing license headers.

Reply:
Oops.  This file doesn't belong in the commit.